yaksh/send_emails.py

```diff
@@ -58,3 +61,30 @@ def send_user_mail(user_mail, key):
         success = False
 
     return success, msg
+
+def send_bulk_mail(subject, email_body, recipients, attachments):
+    try:
+        text_msg = "Yaksh"
+        msg = EmailMultiAlternatives(subject, text_msg, settings.SENDER_EMAIL,
+                                    recipients
+                                    )
+        msg.attach_alternative(email_body, "text/html")
+        if attachments:
+            for file in attachments:
+                path = default_storage.save('attachments/'+file.name,
```

Comment:
It automatically finds the path from MEDIA_ROOT from settings and will store files in that directory